### PR TITLE
C-ICAP: fix typo for maxsize value

### PR DESCRIPTION
**Pull request by mimugmail**
Entry will not be set.

### CODE REVIEW

**Comment by fichtner** [comment]
thanks!

**Comment by mimugmail** [comment]
@fichtner If this needs a version bump please do it, my fork is way too old :/

**Comment by fichtner** [comment]
will do

File: www/c-icap/src/opnsense/service/templates/OPNsense/CICAP/virus_scan.conf [hunk 1]
```diff
@@ -19,7 +19,7 @@ virus_scan.Allow204Responces on
 {% if helpers.exists('OPNsense.cicap.antivirus.passonerror') and OPNsense.cicap.antivirus.passonerror != '' %}
 virus_scan.PassOnError off
 {% endif %}
-{% if helpers.exists('OPNsense.cicap.antivirus.cmaxobjectsize') and OPNsense.cicap.antivirus.maxobjectsize != '' %}
+{% if helpers.exists('OPNsense.cicap.antivirus.maxobjectsize') and OPNsense.cicap.antivirus.maxobjectsize != '' %}
 virus_scan.MaxObjectSize  5M
```
Comment (fkling) [inline www/c-icap/src/opnsense/service/templates/OPNsense/CICAP/virus_scan.conf]: I could be wrong (and sorry for hijacking this PR), but shouldn't this line be something like 

```
virus_scan.MaxObjectSize {{ OPNsense.cicap.antivirus.maxobjectsize }}
```

similar to line 7? If yes, all the other options have to be fixed as well. At the moment everything seems to be hardcoded.

Comment (fichtner) [reply www/c-icap/src/opnsense/service/templates/OPNsense/CICAP/virus_scan.conf]: yes, you are right:

https://github.com/opnsense/plugins/blob/master/www/c-icap/src/opnsense/mvc/app/models/OPNsense/CICAP/Antivirus.xml#L40-L43

**Comment by mimugmail** [comment]
Whoopsie .. this needs more review, sorry :(

**Comment by mimugmail** [comment]
Puh, virus_scan.conf is a mess .. seems I need a sabbatical week from OPNsense :/

**Pull request merged**